containers/Connector/Connector.tsx

```diff
@@ -160,13 +163,31 @@ const useConnector = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [isAppReady]);
 
+	useEffect(() => {
```

Comment:
Not sure if this belongs in the `connector.tsx`. Should be local to wherever we render wallet addresses and logic should probably be in a reusable hook.

Reply:
i understand the intent to turn it into a reusable hook, but if i do i expect issues since the logic does NOT return anything

it merely sets the state variable